OpenTrons/opentrons-api
# feat(app): add welcome modal for unboxing flow

# Overview
Add welcome modal to robot dashboard for unboxing flow. For this, remove updateConfigValue from Confirmation screen and move it to the modal.
Now, when the modal shows on the touchscreen, tapping `Got it` in the modal/ tapping somewhere in the touchscreen is the trigger to complete the unboxing flow.   

close RCORE-759


# Test Plan
1. Change config.json `unfinishedUnboxingFlowRoute: '/robot-settings/rename-robot'` (on-device mode/dev kit)
2. Open RobotDashboard
    The modal is showing up
3. Click/tap `Got it`
4. Check config.json `unfinishedUnboxingFlowRoute` is updated to `null`


# Changelog
- add a modal and tests
- remove updateConfigValue from confirm component and update tests
- add a new image for the modal



# Review requests



# Risk assessment
